core_backend/Makefile

```diff
@@ -23,8 +23,8 @@ fresh-env :
 
 	if [ "$(psycopg2-binary)" = "true" ]; then \
 		$(CONDA_ACTIVATE) $(PROJECT_NAME); \
-		pip uninstall -y psycopg2; \
-		pip install psycopg2-binary==2.9.7; \
+		pip uninstall -y psycopg2==2.9.9; \
+		pip install psycopg2-binary==2.9.9; \
```

Comment:
What does this do?

Reply:
This bit is for those of us for whom psycopg2 doesnt work, but psycopg2-binary does - i.e. me and sid. Removes that lib and replaces it with the binary version (which is fine for development). It's also in the dev docs (but v reasonable not to see it) [here](https://arc.net/l/quote/qphmhbmm)